src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/CommitmentPlans/MoveAzureMLCommitmentAssociation.cs

```diff
@@ -0,0 +1,54 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Management.MachineLearning.CommitmentPlans.Models;
+
+namespace Microsoft.Azure.Commands.MachineLearning
+{
+    [Cmdlet(VerbsCommon.Move, CommitmentPlansCmdletBase.CommitmentAssociationCommandletSuffix)]
```

Comment:
@ericwill-msft does this cmdlet make changes or have side effects? Consider implementing `ShouldProcess` like you do in the cmdlets below